inc/ruleticket.class.php

```diff
@@ -230,6 +230,10 @@ function executeActions($output, $params, array $input = []) {
                            "items_id" => $output['id']]);
                      }
                   }
+                  
+                  if ($action->fields["field"]=='_groups_id_assign' && array_key_exists('_additional_groups_assigns', $output)) {
+                     unset($output['_additional_groups_assigns']);
+                  }
```

Comment:
Proposed behaviour seems correct (I would like a confirmation from someone; ping @flegastelois @orthagh ), but I think the fix should be global (i.e., behaviour should be the same for users / groups / suppliers and for requesters / observers / assigned).

Also, if proposal is accepted, we have to add some tests on this.

```suggestion
                  $actions = $this->getActions();
                  $append_key = $actions[$action->fields["field"]]["appendto"] ?? null;
                  if ($append_key !== null && array_key_exists($append_key, $output)) {
                     unset($output[$append_key]);
                  }
```

Reply:
> I would like a confirmation from someone; ping @flegastelois @orthagh

I agree

> behaviour should be the same for users / groups / suppliers and for requesters / observers / assigned

also agree, for actions where you can choose between appending or assigning